buildstockbatch/schemas/v0.3.yaml

```diff
@@ -45,7 +45,7 @@ gcp-job-environment-spec:
   use_spot: bool(required=False)
 
 gcp-postprocessing_environment-spec:
-  cpus: int(min=1, max=224, required=False)
+  cpus: int(min=1, max=8, required=False)
```

Comment:
Add a link to where these value came from (...google.cloud.run_v2.types.ResourceRequirements)?